cypress/plugins/index.js

```diff
@@ -73,6 +79,56 @@ const dropAndSeedPostgres = async () => {
   if (data.status === "error") {
     throw new Error(data.message);
   }
+  pgDbName = data.dbName;
+}
+
+const replaceDbNameInPgConnectionString = (connectionString, dbName) => {
+  if (!/^postgres:\/\/.*\/[^/]+$/.test(connectionString)) {
+    throw `Incorrectly formatted connection string or unrecognized connection string format: ${connectionString}`;
+  }
+  const lastSlash = connectionString.lastIndexOf('/');
+  const withoutDbName = connectionString.slice(0, lastSlash);
+  return `${withoutDbName}/${dbName}`;
+}
+
+const associateLoginTokenMongo = async (config, {user, loginToken}) => {
+  if (!dbConnection) {
+    dbConnection = new MongoClient(config.env.TESTING_DB_URL);
+    await dbConnection.connect();
+  }
+  const db = await dbConnection.db();
+  await db.collection("users").updateOne({_id: user._id}, {
+    $addToSet: {
+      "services.resume.loginTokens": {
+        when: new Date(),
+        hashedToken: hashLoginToken(loginToken),
+      },
+    },
+  });
+}
+
+const associateLoginTokenPostgres = async (config, {user, loginToken}) => {
```

Comment:
All the Cypress scaffolding is Javascript but the query builder's Typescript 😢.

There's probably some way to hack it, but it's more effort than it's worth.

(Also, this might just be easier anyway)